get_trust_anchor.py

```diff
@@ -375,16 +374,17 @@ def export_ksk(valid_ksks, ds_record_filename, dnskey_record_filename):
         ds_record_contents = ". IN DS {keytag} {alg} 2 {sha256ofkey}".format(\
             keytag=this_key_tag, alg=this_matched_ksk["a"],\
             sha256ofkey=hash_as_hex)
+        print("Writing out {}.".format(ds_record_filename))
         write_out_file(ds_record_filename, ds_record_contents)
 
 
 def main():
     """Main function"""
 
     # Where the files we create are kept
-    trust_anchor_filename = "root-anchors.xml"
-    signature_filename = "root-anchors.p7s"
-    icann_ca_filename = "icanncacert.pem"
+    trust_anchor_filename = tempfile.mktemp()
```

Comment:
tempfile.mktemp is deprecated, are you sure you want to use that?
either way, we might want to set a prefix so that we know what file is what if the user decides to keep them around.